lib/vquic/quiche.c

```diff
@@ -410,6 +412,9 @@ static CURLcode flush_egress(struct connectdata *conn, curl_socket_t sockfd,
     socksent = send(sockfd, out, sent, 0);
     if(socksent < 0) {
       if(errno == EAGAIN || errno == EWOULDBLOCK || errno == EINTR) {
+        FD_ZERO(&fds_write);
+        FD_SET(sockfd, &fds_write);
+        Curl_select(sockfd + 1, NULL, &fds_write, NULL, 0);
```

Comment:
This should not select() by itself since this makes a blocking behavior that excludes all other concurrent transfers while this waits there. We have a system to return EAGAIN and get back to operation when the socket becomes writable again and we should rather make an effort that h3 code can use this as well.

Reply:
OK, now I return CURLE_AGAIN when the sockfd here is blocking.